human-biomolecular-data/data-sources.md

```diff
@@ -19,93 +25,134 @@ training:
 
-<!--- Example of topic, optional --->
-
-Under what licenses do the data sources work?
+However, the benefits of sharing human biomolecular data outweigh the risks. Access to this data is crucial for scientific progress and medical advancements. It's important for us to continue finding ways to responsibly and securely share this valuable resource. Let's work together to make sure that we do so while respecting everyone's rights and ensuring that data is kept safe and secure.
 
 ### Considerations
 
-Using a bullet point style list format as much as possible, describe the considerations for your topic that are specific to infectious diseases within the category (e.g. pathogen characterisation) that you have selected.
+When looking for solutions to human biomolecular data access, you should consider the following aspects:
+- Check with your institution that you are following the ethical and legal requirements.
+- Deposit your data in a controlled access repository
+- Ensure that the data use conditions are publicly available to ease the data access process
+- Assemble your Data Access Committee to process data access requests
 
-Please avoid replicating 'generic' guidelines, i.e. those not specific to infectious diseases, here. Add links to RDMkit in the metadata above, if any are needed. 
+When looking for solutions to data transfer, you can check [this](https://rdmkit.elixir-europe.org/data_transfer) documentation.
 
 ### Existing approaches
 
-Mention existing appreaches related to the topic.
+- You can check a list of existing controlled access repositories:
+  - [European Genome-phenome Archive](https://ega-archive.org/)
+  - [Estonian BioBank](https://genomics.ut.ee/en/content/estonian-biobank)
+  - [Dutch Covid-19 Data Portal](https://covid19initiatives.health-ri.nl/p/Dashboard) 
+  - [PANTHER](https://pantherstudy.org.uk/) 
+  - [ACE](https://co-connect.ac.uk/ace-cohort/)
+- You can use one of these standards to make your data use conditions publicly available to possible data requesters.
+  - The [GA4GH Data Use Ontology DUO](https://github.com/EBISPOT/DUO) is an international standard, which provides codes to represent data use restrictions for controlled access datasets.
+  - The [Automatable Discovery and Access Matrix (ADA-M)](https://github.com/ga4gh/ADA-M) provides a standardised way to unambiguously represent the conditions related to data discovery and access. 
+  - By depositing your data to one of the existing controlled access repositories, they will already show the data use conditions (e.g. [EGAD00001007777](https://ega-archive.org/datasets/EGAD00001007777))
+- A data access committee (DAC) is a group responsible for reviewing and approving requests for access to sensitive data, such as human biomolecular data. Its role is to ensure that requests are in compliance with relevant laws and regulations, that data is being used for legitimate scientific purposes, and that privacy and security are being maintained. 
```

Comment:
I don't think this is an existing approach per se, but rather a useful definition. I would at least cite a DAC, or an example of some and how it was formed.

Reply:
LGTM